### PR TITLE
Warrior lunge/toss nerf

### PR DESCRIPTION
**Pull request by BlueHNT**
## About The Pull Request
Slight nerf to warrior's lunge and toss

## Why It's Good For The Game
Warrior as of now is "busted" and his lunge and fling play the biggest role in this. Slight nerf to his overly giant lunge {whole viewable surrounding} and fling which has the same fling send off as shrike with added stun.

## Changelog
:cl:
balance: rebalanced something
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm [hunk 1]
```diff
@@ -106,7 +106,7 @@
 	RegisterSignal(X, COMSIG_MOVABLE_POST_THROW, .proc/clean_lunge_target)
 	succeed_activate()
 	add_cooldown()
-	X.throw_at(get_step_towards(A, X), 6, 2, X)
+	X.throw_at(get_step_towards(A, X), 4, 4, X)
```
Comment (ZephyrTFA) [inline code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm]:
```suggestion
	//original: X.throw_at(get_step_towards(A, X), 6, 2, X)
	X.throw_at(get_step_towards(A, X), 4, 4, X) // Skyrat Edit - Warrior Toss Nerf
```

File: code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm [hunk 2]
```diff
@@ -256,9 +256,9 @@
 /datum/action/xeno_action/activable/toss/use_ability(atom/A)
 	var/mob/living/carbon/xenomorph/X = owner
 	var/atom/movable/target = owner.pulling
-	var/fling_distance = 5
+	var/fling_distance = 3
```
Comment (ZephyrTFA) [inline code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm]:
```suggestion
	var/fling_distance = 3 // Skyrat Edit - Warrior Toss Nerf - original: 5
```

File: code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm [hunk 3]
```diff
@@ -256,9 +256,9 @@
...
 	var/stagger_slow_stacks = 3
-	var/stun_duration = 1 SECONDS
+	var/stun_duration = 0.5 SECONDS
```
Comment (ZephyrTFA) [inline code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm]:
```suggestion
	var/stun_duration = 0.5 SECONDS // // Skyrat Edit - Warrior Toss Nerf - original: 1 SECONDS
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]
Needs a few changes, in particular it needs to conform to Skyrat Standards.
All the changes that are needed I requested.

**Comment by BlueHNT** [comment]
> Needs a few changes, in particular it needs to conform to Skyrat Standards.
> All the changes that are needed I requested.

If you're using a github like VSC lense, then it already says that...?

**Comment by Smudged** [comment]
Good Change, Warrior has perhaps been a liittlee stronger as of late, due to multiple buffs prior back when warrior was.. nerfed to the ground.

This should help in at least one of the problems marines complain about, and it's more reasonable

**Comment by ZephyrTFA** [comment]
I want to merge this; but you aren't documenting the changes correctly.
Please document the changes correctly.

**Review by BlueHNT — COMMENTED** [review]
Documentation has been added and commited, awaiting approval

**Review by ZephyrTFA — APPROVED** [review]

**Pull request merged**